microbench/edit_scalability.cpp

```diff
@@ -74,52 +76,52 @@ int main(int argc, char const* argv[]) {
 
   // validate args
   if (ingest_threads == 0)
-    throw runtime_error("ingest threads must be greater than zero");
+    throw std::runtime_error("ingest threads must be greater than zero");
   if (algo_threads == 0)
-    throw runtime_error("algo threads must be greater than zero");
+    throw std::runtime_error("algo threads must be greater than zero");
 
-  unique_ptr<scea::graph::MutableGraph> graph;
+  std::unique_ptr<scea::graph::MutableGraph> graph;
   switch (graph_type) {
   case GraphType::lscsr: {
-    graph = make_unique<scea::graph::LS_CSR>(num_vertices);
+    graph = std::make_unique<scea::graph::LS_CSR>(num_vertices);
     break;
   }
   case GraphType::morph: {
-    graph = make_unique<scea::graph::MorphGraph>(num_vertices);
+    graph = std::make_unique<scea::graph::MorphGraph>(num_vertices);
     break;
   }
   default:
-    throw runtime_error("unknown graph_type");
+    throw std::runtime_error("unknown graph_type");
```

Comment:
this is a bit dangerous. Throwing exceptions are only allowed at the highest scope.

Reply:
we should let the author of these changes address them, this is also at the highest scope